manifests/server/tablespace.pp

```diff
@@ -5,20 +5,25 @@
 # @param owner Specifies the default owner of the tablespace.
 # @param spcname Specifies the name of the tablespace.
 # @param connect_settings Specifies a hash of environment variables used when connecting to a remote server.
-# @param port the port of the postgresql instance that sould be used.
+# @param user Sets the OS user to run psql
+# @param group Sets the OS group to run psql
+# @param psql_path Sets path to psql command
+# @param module_workdir
+#   Specifies working directory under which the psql command should be executed.
+#   May need to specify if '/tmp' is on volume mounted with noexec option.
+# @param port Port to use when connecting.
```

Comment:
you deleted the documentation in line 7 and added a different one here, please undo that